nx-X11/programs/Xserver/hw/nxagent/NXdispatch.c

```diff
@@ -1512,7 +1518,9 @@ ProcConvertSelection(register ClientPtr client)
 					CurrentSelections[i].pWin))
 #endif
 	    )
+
 	{        
+	    memset(&event, 0, sizeof(xEvent));
```

Comment:
I just had a quick glance at it: This code generates TWO events and therefore must clear the memory twice!